dev/NavigationView/TestUI/NavigationViewPage.xaml

```diff
@@ -80,6 +80,13 @@
                             IsChecked="{Binding IsPaneOpen, ElementName=NavView, Mode=TwoWay}"
                             Margin="5"/>
 
+                        <CheckBox
+                            x:Name="IsPaneVisibleCheckBox"
+                            AutomationProperties.Name="IsPaneVisibleCheckBox"
+                            Content="IsPaneVisible"
+                            IsChecked="{Binding IsPaneVisible, ElementName=NavView, Mode=TwoWay}"
+                            Margin="5"/>
+
```

Comment:
Is there an interaction test to exercise this?

Reply:
Added one.